### PR TITLE
Add myself as a lead

### PR DESCRIPTION
**Pull request by sttts**
Having initiated 3 and owning 4 of the 6 proposed subprojects, I add myself as a lead.

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -71,11 +71,10 @@ aliases:
     - dklyle
     - rjmorse
   sig-platform-dev-leads:
-    - jzelinskie
```
Comment (enisoc) [inline OWNERS_ALIASES]: These duplicates were put there by the generator. If you run `make generate` again, does it put them back or leave them out? I don't want to drift from whatever the generator currently enforces, although we should also fix the generator to not add duplicates.

**Review by enisoc — COMMENTED** [review]
LGTM as long as `make generate` was run.

cc @jzelinskie @pwittrock

**Comment by jzelinskie** [comment]
LGTM excluding the issue @enisoc mentioned with the generator.

**Comment by sttts** [comment]
Fixed the generator problem.

**Pull request merged**